### ISSUE TITLE
Spells - damage formula

### ISSUE THREAD

**Issue by BlackWand1** [Rughalt/D35E#749]
The spells that does have a damage formula usually use the "min", which would mean the lowest possible caster level.
Maybe this requires a larger redesign. And instead of the min/max have a "caster level" field on items, and Staffs has no limit to go up.

* Scrolls - lowest possible, but can be created with higher CL.
* Wands - same as above.
* Staffs - Highest possible (as the wielder) but the staff itself has a minimum caster level. Not on a per spell level.
* Spells in the head(spellbok) - Same as staffs.

![image](https://user-images.githubusercontent.com/85252943/131262438-4943b5e7-6157-4ed2-86b9-2f447a781474.png)

**Comment by Rughalt** [comment]
I do not understand. Min in this case means that this is the minimum value between `@cl` and 10 and this caps maximum damage at 10d6 which is correct behaviour of this and many spells.

**Comment by Rughalt** [comment]
The staff's issue will be addressed in #748 as for CL level selection when creating potions and wands they are covered by ticket #503

**Comment by BlackWand1** [comment]
Cool.

As for your confusion about this issue.

I notuced that all(most) spells used the "min(cl,10)d6" and my understanding from testing is that min means minimum casterlevel. When spell was converted to scroll the fireball was "converted" to static caster level 5 (5d6).

Same when adding to staffs (which is incorrect).

I tried clarify that for scrolls "min" is correct. Also for wand and potion.

But for staff and memorized caster level should be "max" possible.

But it seems you got them fixed in other fixes :D
